resources/views/components/language-changer.blade.php

```diff
@@ -3,26 +3,31 @@
 ])
 
 @if (count($model->languages) > 1)
-    <x-expander level="2" :summary="__('Change page language')">
-        <h3>{{ __('Available languages') }}</h3>
-        <ul role="list">
-            @foreach ($model->languages as $code)
-                @if (in_array($code, config('locales.supported')))
-                    {{-- Make sure at least the model name is translated to avoid 404 errors. --}}
-                    @if (!$model->isTranslatableAttribute($modelName) || !empty($model->getTranslation($modelName, $code, false)))
+    <div class="flex justify-between">
+        <div class="flex">
+            <label class="whitespace-nowrap" for='available_languages'>{{ __('Page also available in:') }}</label>
+            <ul class="flex flex-wrap" id='available_languages' role="list">
+                @foreach ($model->languages as $code)
+                    @if (in_array($code, config('locales.supported')))
+                        {{-- Make sure at least the model name is translated to avoid 404 errors. --}}
+                        @if (!$model->isTranslatableAttribute($modelName) || !empty($model->getTranslation($modelName, $code, false)))
+                            <li class="ml-2"><a
+                                    href="{{ localized_route($model->getRoutePrefix() . '.show', $model, $code) }}">{{ get_language_exonym($code) }}</a>
+                            </li>
+                        @endif
+                    @else
                         <li><a
-                                href="{{ localized_route($model->getRoutePrefix() . '.show', $model, $code) }}">{{ get_language_exonym($code) }}</a>
+                                href="{{ localized_route($model->getRoutePrefix() . '.show', [Str::camel(class_basename($model)) => $model, 'language' => $code], get_written_language_for_signed_language($code)) }}">{{ get_language_exonym($code) }}</a>
                         </li>
                     @endif
-                @else
-                    <li><a
-                            href="{{ localized_route($model->getRoutePrefix() . '.show', [Str::camel(class_basename($model)) => $model, 'language' => $code], get_written_language_for_signed_language($code)) }}">{{ get_language_exonym($code) }}</a>
-                    </li>
-                @endif
-            @endforeach
-        </ul>
-        <h3>{{ __('Don’t see the language you need?') }}</h3>
-
-        <p><a href="#TODO">{{ __('Contact support') }}</a></p>
-    </x-expander>
+                @endforeach
+            </ul>
+        </div>
+        <div>
+            <p class="whitespace-nowrap">
+                <a href="#TODO">{{ __('Contact support') }}</a>
```

Comment:
Can you add an `id` of `contact` and `tabindex="-1"` to the div in the footer component which contains the `<h2>Contact</h2>`? Then you can change the `href` of this link to `#contact` and the page will scroll to the footer and highlight the contact methods.